src/main/java/net/coderbot/iris/fantastic/FabulousInterface.java

```diff
@@ -0,0 +1,5 @@
+package net.coderbot.iris.fantastic;
```

Comment:
Wrong package?

Reply:
Tbh I thought the "fantastic" was an intentional funny name... but yea it's fabulous right?